Cabal/tests/PackageTests/ConfigureComponent/Exe/Exe.cabal

```diff
@@ -0,0 +1,17 @@
+name:                Exe
+version:             0.1.0.0
+license:             BSD3
+author:              Edward Z. Yang
+maintainer:          ezyang@cs.stanford.edu
+build-type:          Simple
+cabal-version:       >=1.10
+
+executable goodexe
+  main-is:             Good.hs
+  build-depends:       base
+  default-language:    Haskell2010
+
+executable badexe
+  main-is:             Bad.hs
+  build-depends:       totally-impossible-dependency-to-fill == 10000.25.6
```

Comment:
So this exe is deliberately not used right? It's checking that we can build the good one even though the other one cannot be configured. Ok, fine. A comment here to this effect (either here or in the test code) would make this test clearer.